.travis.yml

```diff
@@ -1,4 +1,13 @@
+sudo: false
+git:
+  depth: 1
+branches:
+  only:
+    - master
 language: node_js
-
 node_js:
-  - "0.10"
+  - "4.4"
```

Comment:
Technically, we can leave this at 4. nvm automatically uses the latest in the series. We just needed to mention 4.4+ in the README so that someone doesn't mistaking use `nvm use 4.0.0`. On the other hand, setting this explicitly to the minimum version under which the tests runs does ensure we always test exactly that version. So I'm not sure what the right answer is here. Just wanted to mention that.